tile-rendering/src/main/java/com/oculusinfo/tile/rendering/transformations/tile/FilterByTimeTileTransformer.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2015 Uncharted Software. 
+ * http://www.oculusinfo.com/
+ * 
+ * Released under the MIT License.
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is furnished to do
+ * so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.oculusinfo.tile.rendering.transformations.tile;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import com.oculusinfo.binning.TileData;
+import com.oculusinfo.binning.TileIndex;
+import com.oculusinfo.binning.impl.DenseTileData;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+
+/** 
+ * 	This transformer will take in JSON or tileData object representing bins of a double array  
+ * 		tile and will filter out all counts in the bins representing the time buckets indicated
+ * 		in the configuration.  The double arrays passed back will be in the order that they are 
+ * 		sequenced in the JSON/Tile array passed in.
+ * 
+ */
+
+public class FilterByTimeTileTransformer<T> implements TileTransformer<List<T>> {
```

Comment:
Since this transformer deals only with buckets, would it be better rename it to a more generic a FilterByBucketTileTransformer and rename the args to startBucket, endBucket, and bucketCount?